nntrainer/src/model_loader.cpp

```diff
@@ -307,21 +331,25 @@ int ModelLoader::loadFromIni(std::string ini_file, NeuralNetwork &model,
     if (strncasecmp(dataset_str, sec_name.c_str(), dataset_len) == 0)
       continue;
 
+    /** Parse all the layers defined as sections in order */
+    std::shared_ptr<Layer> layer;
+
     /**
      * If this section is a backbone, load backbone section from this
      * @note The order of backbones in the ini file defines the order on the
      * backbones in the model graph
      */
     const char *backbone =
       iniparser_getstring(ini, (sec_name + ":Backbone").c_str(), unknown);
-    if (backbone != unknown) {
-      loadBackboneConfigIni(ini, backbone, model, sec_name);
+    if (backbone == unknown) {
+      status = loadLayerConfigIni(ini, layer, sec_name);
```

Comment:
IMO, backbone is also layer type so that, we can handle like one of the layer. I mean instead of add loadBAckbonConfigIni separately, we could do this inside loadLayerConfigIni with LayerType keyword.

Reply:
With loadLayerConfigIni, the layer is always added to the model. This is not desired for backbone with ini. Further with ini backbone, with more support to come for setting scaling factors, etc, lets keep it separate for now, and will merge once all features are supported.
For backbone with external frameworks, `loadBackboneConfigExternal` is actually just a wrapper for `loadLayerIni` which sets the modelfile property.

Reply:
Both the loadConfig for backboneExternal and Layer can be merged if we update the ini style.
Current style:
```ini
[block1]
backbone: tmp.ini/tflite
```

New style
```ini
[block1]
type: backbone
modelfile: tmp.ini/tflite
```

However, IMO, first is more direct and intuitive.